DEVELOPER_GUIDE.md

```diff
@@ -175,7 +170,7 @@ For example, reindex requires the `connect` permission so it can perform reindex
 
 ### `plugins`
 
-Officially supported plugins to OpenSearch. We decide that a feature should be a plugin rather than shipped as a module because we feel that it is only important to a subset of users, especially if it requires extra dependencies.
+Built-in plugins to OpenSearch. We decide that a feature should be a plugin rather than shipped as a module because we feel that it is only important to a subset of users, especially if it requires extra dependencies.
```

Comment:
Correct me if I'm wrong but I believe these plugins not built-in into the distributions and users have to install these manually.

Reply:
You're right. Not sure why I changed this, reverting.